en_US/changes/changes-ce-v5.md

```diff
@@ -24,6 +24,9 @@
 
 - [#12606](https://github.com/emqx/emqx/pull/12606) The Prometheus API experienced crashes when the specified SSL certificate file did not exist in the given path. Now, when an SSL certificate file is missing, the `emqx_cert_expiry_at` metric will report a value of 0, indicating the non-existence of the certificate.
 
+- [#12620](https://github.com/emqx/emqx/pull/12620) Fixed an issue when sensitive headers for HTTP connector may be printed in the `debug` level log.

```

Comment:
This one seems to describe a similar issue as in #12542. Are they duplicated?

Reply:
@Meggielqk Good point. #12620 is a more general fix for the same issue. I suggest to leave just that one in the changelog and remove #12542.